daemons/gptp/common/ieee1588port.cpp

```diff
@@ -941,7 +945,7 @@ void IEEE1588Port::becomeSlave( bool restart_syntonization ) {
   clock->deleteEventTimer( this, ANNOUNCE_INTERVAL_TIMEOUT_EXPIRES );
   clock->deleteEventTimer( this, SYNC_INTERVAL_TIMEOUT_EXPIRES );
 
-  port_state = PTP_SLAVE;
+    port_state = PTP_SLAVE;
```

Comment:
This file has 3 different indenting formats: 2 spaces, 4 spaces and tabs. For this particular line, it is two spaces, so I kept it like that.